spark-operator/src/main/java/org/apache/spark/k8s/operator/config/SparkOperatorConfManager.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.k8s.operator.config;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.Properties;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Loads ConfigOption from properties file. In addition, loads hot properties override from config
+ * map if dynamic config is enabled.
+ */
+@Slf4j
+public class SparkOperatorConfManager {
+  public static final String INITIAL_CONFIG_FILE_PATH =
+      "/opt/spark-operator/conf/spark-operator.properties";
```

Comment:
To be free from the base directory, we had better define `/opt/spark-operator` as an independent constant and use it.